e2e-tests/native/test/specs/navigateToLeafCategory.e2e.ts

```diff
@@ -18,6 +18,7 @@ describe('navigate to leaf category', () => {
     await leafCategory.click()
 
     const leafCategoryContent = $(new Selector().ByContainedText('you will learn German').build())
-    await leafCategoryContent.waitForDisplayed()
+    await leafCategoryContent.waitForDisplayed({ timeout: 20000 })
```

Comment:
seems like it should wait explicitly for it to be displayed in new version according to the [documentation](https://webdriver.io/docs/api/element/isDisplayed/)

> As opposed to other element commands WebdriverIO will not wait for the element to exist to execute this command.